test/disconnect-spec.js

```diff
@@ -0,0 +1,84 @@
+var docker = require('./docker');
+var assert = require('assert');
+var producer = require('../src/index')().producer;
+var consumer = require('../src/index')().consumer;
+var utils = require('../src/modules/utils');
+
+describe('disconnections', function() {
+  before(function() {
+    this.timeout(0);
```

Comment:
is this needed, can it be set to some more realistic value (albeit large) to make sure testes finish in a reasonable time.